cli/slsa-verifier/main_test.go

```diff
@@ -838,6 +839,7 @@ func Test_runVerifyGCBArtifactImage(t *testing.T) {
 		source         string
 		pBuilderID     *string
 		outBuilderID   string
+		outProvenance  string
```

Comment:
this is unused until the TODO?

Reply:
oops, I forgot to remove this. You're correct. Let me delete this field.